client/src/components/Billing/checkout.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { CardElement, injectStripe } from 'react-stripe-elements';
+import axios from 'axios';
+
+class CheckoutForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { complete: false };
+        this.submit = this.submit.bind(this);
+    }
+
+    async submit(ev) {
+        let { token } = await this.props.stripe.createToken({ name: 'Name' });
+        let response = await axios.post('http://localhost:5000/charge', {
```

Comment:
You are hardcoding localhost here. Is this intended?

Reply:
Yes